src/SFA.DAS.EmployerIncentives.Domain/IncentiveApplications/IncentiveApplication.cs

```diff
@@ -79,5 +74,10 @@ private void AddApprenticeship(Apprenticeship apprenticeship)
             _apprenticeships.Add(apprenticeship);
             Model.ApprenticeshipModels.Add(apprenticeship.GetModel());
         }
+
+        public void CalculateEarnings()
```

Comment:
I don't understand why his method has been put on the domain object?  No state change to the domain object or business rule validation is being done.  Why not just get the command handler to publish the CalculateEarningsCommand ?